packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -378,9 +442,38 @@ class SemanticsNode extends AbstractNode {
     }
   }
 
+  /// Returns a summary of the semantics for this node.
+  ///
+  /// If this node has [mergeAllDescendantsIntoThisNode], then the returned data
+  /// includes the information from this node's descendants. Otherwise, the
+  /// returned data matches the data on this node.
+  SemanticsData getSemanticsData() {
+    int flags = _flags;
+    int actions = _actions;
+    String label = _label;
+
+    if (mergeAllDescendantsIntoThisNode) {
+      _visitDescendants((SemanticsNode node) {
+        flags |= node._flags;
+        actions |= node._actions;
+        if (label.isEmpty)
+          label = node._label;
```

Comment:
shouldn't we merge the strings? I thought I merged the strings separated by newlines or something.